core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1322,4 +1377,52 @@ object Chunk {
     private val CharClass       = classTag[Char]
     private val CharClassBox    = classTag[java.lang.Character]
   }
+
+  object Byte {
```

Comment:
Is there a reason for all the subtypes to live in these `Byte/Char/Int` objects?